src/wallet/wallet.h

```diff
@@ -248,6 +248,28 @@ class CWalletTx : public CMerkleTx
     const CWallet* pwallet;
 
 public:
+    /**
+     * Key/value map with information about the transaction.
+     *
+     * The following keys can be read and written through the map and are
+     * serialized in the wallet database:
+     *
+     *     "comment", "to"   - comment strings provided to sendtoaddress,
+     *                         sendfrom, sendmany wallet RPCs
+     *     "from", "message" - obsolete fields that could be set in UI prior to
+     *                         2011 (removed 4d9b223)
```

Comment:
4d9b223 is the commit hash? github's UI can't find it. If it is commit hash, please add by/in before the ID.